mailpoet/lib/Subscription/Throttling.php

```diff
@@ -34,7 +34,9 @@ public function throttle() {
       if (!empty($subscriberIp)) {
         $subscriptionCount = $this->subscriberIPsRepository->getCountByIPAndCreatedAtAfterTimeInSeconds($subscriberIp, $subscriptionLimitWindow);
         if ($subscriptionCount > 0) {
-          $timeout = $subscriptionLimitBase * pow(2, $subscriptionCount - 1);
+          $timeout = $subscriptionLimitBase * (pow(2, $subscriptionCount - 1));
+          // Cap timeout and avoid float numbers that can happen if $subscriptionCount > 63
```

Comment:
I wouldn't mention the number of subscriptions here. In my tests it was different, maybe this depends on a system.

Reply:
> In my tests it was different, maybe this depends on a system.

True, it depends on `PHP_INT_MAX`. Removed